test/hotspot/jtreg/vmTestbase/gc/gctests/mallocWithGC2/mallocWithGC2.java

```diff
@@ -113,8 +113,6 @@ public void run() {
                                 tArray[i].start();
 
                         tArray[0].join(); // wait for the javaHeapEater Thread to finish
-                        tArray[1].stop(); // Once javaHeapEater is finished, stop the
-                        // the cHeapEater thread.
                 } catch (Exception e) {
                         throw new TestFailure("Test Failed.", e);
                 }
```

Comment:
Drive-by comment on this source file is that it looks like it uses 8-space indent everywhere, maybe tabs were converted to 8 spaces by mistake?

Reply:
there are a lot of old tests with the 8-spaces tab. I don't want to fix it in this bugfix. Makes sense?